mapper-runtime/src/main/java/com/datastax/oss/driver/internal/mapper/DefaultMapperContext.java

```diff
@@ -107,6 +130,11 @@ public NameConverter getNameConverter(Class<? extends NameConverter> converterCl
     return customState;
   }
 
+  @Override
+  public boolean isSchemaValidationEnabled() {
```

Comment:
@olim7t  according to your suggestion:
```
enabling schema validation through an annotation is a bit too static. It would probably be more flexible with a runtime parameter:
new InventoryMapperBuilder(session).setSchemaValidation(true).build()
```
I've moved the check into Builder. I've created a new field `schemaValidationEnabled`  but it obviously breaks the backward compatibility - if that is not possible for the mapper I can put that schemaValidatEnabled field into `customState` and propagate it there. Let me know wdyt

Reply:
solved by using `customState`